src/main/scala/com/gu/hmac/HMACHeaders.scala

```diff
@@ -18,33 +18,42 @@ sealed trait HMACError extends NoStackTrace
 case class HMACInvalidTokenError(message: String) extends HMACError
 case class HMACInvalidDateError(message: String) extends HMACError
 
-case class HMACToken(value: String)
 
+object HTTP extends Enumeration {
```

Comment:
Fixed the typo, thanks. Good point, I think we should have probably done all of this (including the MD5 helper) as feature branches that then merge into another feature branch, not `master`.

Let's have a chat tomorrow, but I think at this point we might as well carry on.